integration-tests/utils/firecloud-utils.js

```diff
@@ -21,7 +19,19 @@ const signIntoFirecloud = async (page, token) => {
    * user-status). Instead of reworking the sign-in logic for a case that (for the most part) only
    * a computer will operate fast enough to encounter, we'll just slow the computer down a little.
    */
+  await page.waitForResponse(response => {
+      return response.url().startsWith('https://accounts.google.com/o/oauth2/') && response.request().method() === 'GET'
+    }, { timeout: 30 * 1000 }
+  )
```

Comment:
Moved up long comments to the beginning. Based on these comments, wait for Google `/oauth2` request to finish before sign in with token.